cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -23,6 +23,52 @@ Cask Data Application Platform Release Notes
    :backlinks: none
    :depth: 2
 
+`Release 3.2.2 <http://docs.cask.co/cdap/3.2.2/index.html>`__
+=============================================================
+
+Improvement
+-----------
+
+- `CDAP-4133 <https://issues.cask.co/browse/CDAP-4133>`__ -
+  Added ability to get the live-info for the CDAP AppFabric system service.
+
+Bug Fixes
+---------
+
+- `CDAP-3644 <https://issues.cask.co/browse/CDAP-3644>`__ -
+  Fixed a problem with upgrading the metadata system datasets.
```

Comment:
Upgrading metadata datasets is not a bug, it is a task. Is improvement section a better place for it?